index.js

```diff
@@ -36,6 +37,18 @@ function stack(results) {
 
 		i++;
 
+		// Don't print the full stack but the only useful line showing
+		// the actual test file stack
+		var dir = path.dirname(module.parent.filename);
```

Comment:
Sorry for my thickness, but what is this *actually* doing? Isn't it just filtering out anything that's not in the project directory? What is this improving/fixing?